src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -278,16 +278,20 @@ def _flexible_server_params(command_group):
 
         with self.argument_context('{} flexible-server delete'.format(command_group)) as c:
             c.argument('server_name', id_part='name', options_list=['--name', '-n'], arg_type=server_name_getter_arg_type)
-            c.argument('force', options_list=['--force'], action='store_true',
-                       help='Delete the server without prompt')
+            c.argument('yes', options_list=['--yes', '-y'], action='store_true',
```

Comment:
renaming parameters name is breaking change. could you explain why we need such renaming?

Reply:
we wanted to deleting align confirmation parameters same as deleting commands of other resources

Reply:
@Juliehzl have updated the PR to reflect these changes.  Our new service was just rolled out last week so usage is still low, so there is relatively low risk.